ssh/README.md

```diff
@@ -2,60 +2,40 @@
 
 SSH client and server running over SCION network.
 
-# Installation
-
-## Prerequisite
+### Usage
 
 SCION infrastructure has to be installed and running. Instructions can be found [here](https://netsec-ethz.github.io/scion-tutorials/)
 
-Additional development library for PAM is needed:
-```
-sudo apt-get install libpam0g-dev
```

Comment:
This libpam is still a requirement for building this ssh app, I'd keep this info here as a reminder.

Reply:
Done.